users/reallyinvincible/Task_3/index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link type="text/css" rel="stylesheet" href="assets/quiz.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
+            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+            crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+            crossorigin="anonymous"></script>
+    <script type="text/javascript" rel="script" src="assets/quiz.js"></script>
+    <title>Quiz</title>
+</head>
+<body>
+<nav class="navbar" id="nav">
+    <span class="navbar-brand mb-0 h1"><h1>Welcome to The XAM, the most <i><b>unusual</b></i> and <i><b>unique</b></i> quiz ever</h1><br>
+        <h5>Get ready to answer some of the wackiest questions ever!</h5></span>
+</nav>
+
+<div class="card">
+    <div class="card-header" style="background: #673AB7; text-align: center; color: aliceblue;">
+        <b>The Emperor Claudius legalized what at banquets?</b>
+    </div>
+    <div class="options" style="background: #B39DDB;">
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions1" id="inlineRadio11" value="0">
+            <label class="form-check-label" form="inlineRadio1">Alcohol Consumption</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions1" id="inlineRadio12" value="0">
+            <label class="form-check-label" form="inlineRadio2">Complete Nudity</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions1" id="inlineRadio13" value="1">
+            <label class="form-check-label" form="inlineRadio3">Farting</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions1" id="inlineRadio14" value="0">
+            <label class="form-check-label" form="inlineRadio3">Burping</label>
+        </div>
+    </div>
+</div>
+
+<div class="card">
+    <div class="card-header" style="background: #3F51B5; text-align: center; color: aliceblue;">
```

Comment:
Include the `css` in the external css file instead

Reply:
Since every single card has a different background color, writing an external css file is equivalent to writing inline.

Reply:
Hi, I would suggest reading more on `css` inline css is a lot different than external css :) Moreover the code is cleaner